@navikt/core/react/src/form/search/Search.tsx

```diff
@@ -161,6 +161,7 @@ export const Search = forwardRef<HTMLInputElement, SearchProps>(
           title={clearButtonLabel || translate("clear")}
           hidden={!showClearButton}
           onClick={(event) => handleClear({ trigger: "Click", event })}
+          type="button"
```

Comment:
A Search component inside of a `<form>` would automatically empty when submitted, because the clear button didn't have a type and was interpreted as a submit button. 😬